tests/unit/services/test_jobserver.py

```diff
@@ -62,6 +72,25 @@ def test_fetch_release_returns_correct_files(fetch_release, fetch_file):
     assert "90.23" in output["common_codes"][0]["Proportion"]
 
 
+def test_fetch_release_handles_missing_column_for_common_codes(
+    fetch_release, fetch_deciles_chart, fetch_event_counts, responses
+):
+    responses.remove(
+        url=urljoin(jobserver.JOB_SERVER_URL, "/api/v2/releases/file/WXC18BDMBTM0M")
+    )
```

Comment:
Does this mean one of the fixtures needs to be dropped?

Reply:
Ah, good spot. I dropped the fixture but forgot to remove this line afterwards, so it's doing nothing at the moment.